### DISCUSSION TITLE
Whitelisting IPs

### DISCUSSION THREAD

**Discussion by HeadStudios** [ash-jc-allen/short-url#197] [Ideas]
Any possibility for whitelisting IP addresses? I want to check that link works without counting visit against it before sending

**Comment by ash-jc-allen** [comment]
Hey @HeadStudios! That's a great suggestion that I think could come in handy. However, I think it might be a bit out of the scope of this package, unfortunately.

I feel like this is something that would be best being implemented directly in your app code. If it was me, I think I'd create a middleware class that checks the IP. You could register this middleware in the `short-url.middleware` config field. Hopefully this helps a bit? 🙂
